addons/mass_mailing/controllers/main.py

```diff
@@ -23,6 +23,13 @@ def _log_blacklist_action(self, blacklist_entry, mailing_id, description):
         model_display = mailing.mailing_model_id.display_name
         blacklist_entry._message_log(body=description + " ({})".format(model_display))
 
+    @http.route(['/company/social/<string:social>'], type='http', website=True, auth="public", sitemap=False)
+    def social(self, social, **kwargs):
```

Comment:
Wait, what ? No !

Reply:
I actually copied the code from https://github.com/odoo/odoo/blob/880954ebfc1106411b7f7a7d60aee05dfae60893/addons/website/controllers/main.py#L265